examples/ui/ui_texture_slice_flip_and_tile.rs

```diff
@@ -0,0 +1,75 @@
+//! This example illustrates how to how to flip and tile images with 9 Slicing in the UI
+
+use bevy::{prelude::*, winit::WinitSettings};
+use bevy_render::texture::{ImageLoaderSettings, ImageSampler};
+
+fn main() {
+    App::new()
+        .add_plugins(DefaultPlugins)
+        .insert_resource(UiScale(2.))
+        // Only run the app when there is user input. This will significantly reduce CPU/GPU use.
+        .insert_resource(WinitSettings::desktop_app())
+        .add_systems(Startup, setup)
+        .run();
+}
+
+fn setup(mut commands: Commands, asset_server: Res<AssetServer>) {
+    let image = asset_server.load_with_settings(
+        "textures/fantasy_ui_borders/numbered_slices.png",
+        |settings: &mut ImageLoaderSettings| {
+            // Need to use nearest filtering to avoid bleeding between the slices with tiling
```

Comment:
That's only because they're pixely, right? Would a more high-res png also require this?

Reply:
It depends on the texture and the result the user wants.  If, as in the `ui_texture_slice` example, the source texture is seamless and the sides are just mono-coloured horizontal and vertical lines then it can be stretched or tiled without any artifacts even with blending. 

With the numbered texture from the example here, there is bleeding between each slice if you use `ImageSampler::linear()`. The same was true of the previous implementation using multiple sprites, it's not a regression at least.